src/tests/JIT/interpreter/CMakeLists.txt

```diff
@@ -5,7 +5,11 @@ include_directories(${INC_PLATFORM_DIR})
 set(SOURCES pinvoke.cpp)
 
 # add the executable
-add_library (pinvoke SHARED ${SOURCES})
+if (STATIC_LIBS_ONLY)
+    add_library (pinvoke STATIC ${SOURCES})
+else ()
+    add_library (pinvoke SHARED ${SOURCES})
+endif ()
```

Comment:
This will eventually need to be copy-pasted to all native test projects that build libraries. I think it would be better to make more succinct, e. g.:
1) Drop the library type in `add_library` and set `BUILD_SHARED_LIBS` to on/off for not-WASM/WASM.
2) Set a property like `TEST_LIB_TYPE` to `SHARED`/`STATIC` and use it here: `add_library (pinvoke ${TEST_LIB_TYPE} ${SOURCES})`.